student.rb

```diff
@@ -0,0 +1,9 @@
+module Student
+  def initialize(classroom)
+    @classroom = classroom
```

Comment:
here you should also initialize ` age `,  `  name = "Unknown" ` and  `parent_permission = true`
and add its super classes 😄